### ISSUE TITLE
traceImportant produces no message in TeamCity

### ISSUE THREAD

**Issue by otto-gebb** [fsharp/FAKE#2257]
Messages logged with `traceImportant` are not displayed in the TeamCity build log.
The word WARNING is logged instead.

### Repro steps

1. Set `TEAMCITY_VERSION` environment variable to 1.
1. Run a FAKE script with a `traceImportant` call in it.

### Expected behavior

The message is logged as

```
##teamcity[message text='your_message_here' status='WARNING']
```

### Actual behavior

The message is logged as

```
##teamcity[message status='your_message_here' text='WARNING']
```

**Comment by BlythMeister** [comment]
The function to fix is in TeamCityIntetnal in Fake.BuildServer.TeamCity

There is all the mapping there, and I can see your right, the message and status variables are going into the string format the wrong way round.

**Comment by BlythMeister** [comment]
If you could raise a PR please that would be great.

**Comment by otto-gebb** [comment]
I will.

**Comment by otto-gebb** [comment]
@matthid would you kindly create a build with the fix of this? It may also fix #2252.

**Comment by matthid** [comment]
Unfortunately it didn't make it into the last release. However all merged PR are immediately released into our [staging environment](https://fake.build/contributing.html#Staging-environment). You can use version version 5.12.2-alpha.905+vsts.ea24360d032d840285c5b5ea419b7d7a1c0b480d

Note that staging builds are not stored forever but only latest two builds are available (ie you builds will break)

@otto-gebb Does that work?

**Comment by otto-gebb** [comment]
@matthid thanks for the suggestion, but I could not get the pre-reease version to work by following the instructions provided. Maybe I did something wrong.
I ran `dotnet tool install -g fake-cli --version 5.12.2-alpha.905+vsts.ea24360d032d840285c5b5ea419b7d7a1c0b480d --add-source https://www.myget.org/F/fake-vsts/api/v3/index.json`, so that `dotnet tool list -g` shows "5.12.2-alpha.905". I also changed paket.dependencies so that all packages have "prerelease" version, but I still observe the old behavior.
